src/targeting.js

```diff
@@ -5,66 +5,113 @@ const bidmanager = require('./bidmanager');
 const utils = require('./utils');
 var CONSTANTS = require('./constants');
 
-var targeting = exports;
 var pbTargetingKeys = [];
 
-targeting.resetPresetTargeting = function(adUnitCode) {
-  if (isGptPubadsDefined()) {
+export function newTargeting() {
```

Comment:
As written, all these instances are hardcoded to depend on two pieces of mutable/global state: `$$PREBID_GLOBAL$$` (through `getBidLandscapeTargeting` and `getAlwaysUseBidTargeting`) and the `bidmanager` (through `getStandardKeys`).

These two things should be arguments here, with the globals passed in in the `export const targeting` line. That will make it possible (for someone else) to clear the unit tests of the issues that @mkendall07 ran into back in #1427, and bring the project one step closer to #1508.